swagger-spec/files/detail.yaml

```diff
@@ -3,17 +3,18 @@ get:
   description: >-
     Retrieves the details of a file (or folder)
 
-    ####Returns
+    #### Returns
 
-    Returns a JSON object with a `data` key containing the representation of the requested
-    file, if the request was successful.
+    Returns a JSON object with a `data` key containing the metadata for the requested file or folder.
+    If a metadata request is issued against a folder path the call will return the return the metadata for the folder's children.
```

Comment:
Suggested change:
> If a metadata request is issued against a folder, the request will return the metadata for every object in the folder.

Prefer "request" to "the call" and I think something other than "children" for describing the items in a folder.